src/v2/components/UserSettings/index.tsx

```diff
@@ -480,6 +481,47 @@ const UserSettings: React.FC<UserSettingsProps> = ({ me, updateAccount }) => {
                   </Field>
                 </>
               )}
+
+              <ContextDivider />
+
+              <Field
+                name="darkmode"
+                initialValue={document.documentElement.dataset.theme === 'dark'}
+              >
+                {props => {
+                  return (
+                    <>
+                      <InputContainer mt={6} mb={5}>
+                        <Label>Darkmode</Label>
+
+                        <RadioOptions
+                          value={props.input.value}
+                          onSelect={value => {
+                            const theme = value === true ? 'dark' : 'default'
+                            toggleOrSetTheme(theme)
+                          }}
+                        >
+                          <RadioOptions.Option value={true}>
+                            {({ selected }) => (
+                              <Text f={4} mb={3} selected={selected}>
+                                Enable Darkmode
```

Comment:
Also minor: change to just "Enable / Disable"

Reply:
I set it to be more verbose here based on the other toggles, which explicitly spell out each option. Assumed it was related to accessibility. Will fix 👍